### PR TITLE
enh(JavaScript): binary-search

### PR DESCRIPTION
**Pull request by underager**
typo in comment

**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [ ] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [x] Optimization in previous algorithms
- [x] Code style update
- [x] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

**Comment by ming-tsai** [comment]
> typo in comment
> 
> **PR Checklist:**
> 
> * [ ]  My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
> * [ ]  My addition is on refer on the language README.md file
> * [ ]  My addition does not have a spelling problem
> * [ ]  My submission has a proper and user-friendly description of the algorithm
> * [ ]  My submission has the time complexity of the algorithm
> * [ ]  My submission has sample input-output of the program (NOT FOR PYTHON)
> 
> **What kind of change does this PR introduce?** (check at least one)
> 
> * [ ]  Bugfix
> * [ ]  New algorithm
> * [ ]  Optimization in previous algorithms
> * [ ]  Code style update
> * [ ]  Refactor
> * [ ]  Documentation
> * [ ]  Other, please describe:
> 
> **Briefly describe the changes in this PR**
> 
> **Other information:**

Please check the pull request checklist

**Comment by underager** [comment]
I have update the PR, please let me know if there is anything else, thank you.

File: algorithms/JavaScript/src/searching/binary-search-recursive.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ function binaryRecursive(arr, low, high, item) {
 }
 
 // returns 3 , Found at Index 3
-console.log(binaryRecursive([1, 3, 5, 7, 8, 9], 0, 5, 7));
+console.log(binaryRecursive([1, 3, 5, 7, 8, 9], 5, 7));
```
Comment (ming-tsai) [inline algorithms/JavaScript/src/searching/binary-search-recursive.js:44]: It doesn't return 3, please check the method
image

Comment (underager) [reply algorithms/JavaScript/src/searching/binary-search-recursive.js:44]: Hi @ming-tsai, i have resolved the above mentioned issue, please check.

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**